subsys/bluetooth/mesh/Kconfig

```
@@ -67,7 +67,7 @@ if BT_MESH_CDB
 
 config BT_MESH_CDB_NODE_COUNT
 	int "Maximum number of nodes in the database"
-	default 1
+	default 8
```

If I remember correctly, CDB can be used without the provisioner function. Do we still need to keep the minimum parameters in this case?

---

That's true, but I don't think it makes sense to treat that as a special case. Also, what is the use case for this, really? 

You wouldn't put CDB on a microbit anyway (although this doesn't prevent anyone from doing so), so IMO usability trumps memory footprint

---

Yes, but it won't be too complicated.

```suggestion
	default 8 if BT_MESH_PROVISIONER
```